cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -88,6 +88,14 @@ message CassandraConfigRole {
     optional TaskConfig cassandraYamlConfig = 11;
     // mesos role to be used to receive resource offers
     optional string mesosRole = 12;
+
+    /**
+     * A pre-defined data directory specifying where cassandra should write it's data.
+     * NOTE:
+     * This field will be removed once MESOS-1554 is released and the framework will
+     * be able to allocate the data volume itself.
+     */
+    optional string preDefinedDataDirectory = 13;
```

Comment:
Can we remove a field from protobuf? Maybe we just don't use it once MESOS-1554 is in a release.